vnext/Microsoft.ReactNative/Views/cppwinrt/ViewPanel.idl

```diff
@@ -27,6 +34,7 @@ namespace Microsoft.ReactNative
     XAML_NAMESPACE.Media.Brush BorderBrush { get; set; };
     XAML_NAMESPACE.CornerRadius CornerRadius { get; set; };
     Boolean ClipChildren { get; set; };
+    Boolean IsAccessibilityHidden { get; set; };
```

Comment:
Is there any reason this has to be exposed via the public interface for ViewPanel? This means that if someone creates a custom view manager based on ViewPanel, they can arbitrarily set IsAccessibilityHidden. We've already seen the trouble with this in previous PRs becoming breaking changes (e.g., for moving overflow: hidden into ViewManagerBase and in the PR that moves ViewPanel from Panel -> Grid).